lib/views/admin_request/_hidden_user_explanation_reasons.html.erb

```diff
@@ -6,6 +6,8 @@
                             state: 'not_foi',
                             message: 'personal_information' %>
 
+<br>
```

Comment:
👎 looks fine without this to me:

![screen shot 2018-11-20 at 17 10 46](https://user-images.githubusercontent.com/282788/48790481-5572fd80-ece7-11e8-8378-38db57d659c8.png)

Reply:
No problem - it's a bodge anyway, we can always revisit it and give the layout proper consideration if it gets hard to read with more option added.

(This was probably done when the labels were a bit longer, and I sometimes have the font size bumped up)